actix-cors/src/lib.rs

```diff
@@ -35,10 +37,14 @@
 //!     Ok(())
 //! }
 //! ```
-//! In this example custom *CORS* middleware get registered for "/index.html"
-//! endpoint.
 //!
-//! Cors middleware automatically handle *OPTIONS* preflight request.
+//! In this example a custom CORS middleware get registered for the
+//! "/index.html" endpoint.
```

Comment:
```suggestion
//! In this example a custom CORS middleware is registered for the
//! "/index.html" endpoint.
```

also move above the code block